packages/hooks/src/useP2PAdvertList.ts

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
-import { usePaginatedFetch } from '@deriv/api';
+import { useFetch } from '@deriv/api';
 import useExchangeRate from './useExchangeRate';
 
 /**
  * This custom hook returns available adverts for use with 'p2p_order_create' by calling 'p2p_advert_list' endpoint
  */
-const useP2PAdvertList = (
-    payload?: NonNullable<Parameters<typeof usePaginatedFetch<'p2p_advert_list'>>[1]>['payload']
-) => {
+const useP2PAdvertList = (payload?: NonNullable<Parameters<typeof useFetch<'p2p_advert_list'>>[1]>['payload']) => {
     const { getRate } = useExchangeRate();
-    const { data, ...rest } = usePaginatedFetch('p2p_advert_list', { payload });
+
+    const limit: number = payload?.limit || 10;
+    const [offset, setOffset] = React.useState<number>(payload?.offset || 0);
```

Comment:
same for offset, since its already default 0 when we call the endpoint

Reply:
Same as the above